app/controllers/api/functions.php

```diff
@@ -229,6 +233,8 @@
             'providerBranch' => $providerBranch,
             'providerRootDirectory' => $providerRootDirectory,
             'providerSilentMode' => $providerSilentMode,
+            'memory' => $memory,
+            'cpus' => $cpus
```

Comment:
The collection itself should contain both CPU and memory attributes as this allows us to be more flexible in the future. so you need to map the string `s-1vcpu-512mb` which comes in as a param to 1 CPU and 512MB memory